js/basemap_selector.ts

```diff
@@ -0,0 +1,104 @@
+import type { RenderProps } from "@anywidget/types";
+import { css, html, PropertyValues, TemplateResult } from "lit";
+import { property, query } from "lit/decorators.js";
+
+import { legacyStyles } from "./ipywidgets_styles";
+import { materialStyles } from "./styles";
+import { loadFonts } from "./utils";
+import { LitWidget } from "./lit_widget";
```

Comment:
Should these imports be alphasorted by filepath?

Reply:
Yes, done.